ui/src/routes/organizations/$orgId/products/$productId/index.tsx

```diff
@@ -59,14 +60,90 @@ import {
   paginationSearchParameterSchema,
   sortingSearchParameterSchema,
 } from '@/schemas';
+import { LastJobStatus } from './-components/last-job-status';
+import { LastRunDate } from './-components/last-run-date';
+import { LastRunStatus } from './-components/last-run-status';
 import { ProductIssuesStatisticsCard } from './-components/product-issues-statistics-card';
 import { ProductPackagesStatisticsCard } from './-components/product-packages-statistics-card';
 import { ProductRepositoriesStatisticsCard } from './-components/product-repositories-statistics-card';
 import { ProductViolationsStatisticsCard } from './-components/product-violations-statistics-card';
 import { ProductVulnerabilitiesStatisticsCard } from './-components/product-vulnerabilities-statistics-card';
+import { TotalRuns } from './-components/total-runs';
+
+const defaultPageSize = 10;
```

Comment:
I think @mmurto suggested that the `defaultPageSize` could be 5 for the repositories table in the overview page, but this can be changed later as well if needed.

Reply:
I changed it already for this PR, thanks for reminding.